src/scripts/browserDeviceProfile.js

```diff
@@ -318,6 +318,13 @@ import browser from './browser';
         }
 
         const isSurroundSoundSupportedBrowser = browser.safari || browser.chrome || browser.edgeChromium || browser.firefox || browser.tv || browser.ps4 || browser.xboxOne;
+        const isAc3Eac3Supported = supportsAc3(videoTestElement) || supportsEac3(videoTestElement);
+
+        // AC3/EAC3 hinted that device is able to play dolby surround sound.
+        if (isAc3Eac3Supported && isSurroundSoundSupportedBrowser) {
+            return 6;
```

Comment:
Can number of channels be more than `6`?
We could revert my previous suggestion and return `speakerCount > 6 ? speakerCount : 6`.

Reply:
The formats do support more than 6 channels, not sure about browsers though.

Reply:
Done.